modules/registration/sitesupport/registration.js

```diff
@@ -15,7 +16,7 @@
 var sidebarMainDiv = 'main_content';
 
 function basicRegistrationRequest(parameter, finish) {
-  basicBackendRequest('POST', 'registration', parameter, finish);
+  basicBackendRequest('GET', 'registration', parameter, finish);
```

Comment:
THis one will be going away soon anyway but WHY?  The request using this is  a POST request.

Reply:
You'd think so, but I got a 500 trying it that way!

Reply:
(This can be reverted if we can figure out what was really going on)

Reply:
Well this is only used by `refreshBadgeData`  and that is setting `refreshData` and if you look in `modules/registration/pages/pre/inc` you can see that `refreshData` is used in `POST` requests.

Now it may not be working... but that would be unrelated to it being POST vs GET. I feel like this change should be removed and in a different patch all that code can be removed.  In fact I ahve an in progress tree that is doing just that.